src/SDKs/ManagedServices/ManagedServices.Test/ManagedServices.Test.csproj

```diff
@@ -0,0 +1,40 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <!-- Please do not move/edit code below this line -->
+  <Import Condition=" Exists('$([MSBuild]::GetPathOfFileAbove(AzSdk.test.reference.props))') " Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
+  <PropertyGroup>
+    <PackageId>ManagedServices.Test</PackageId>
+    <VersionPrefix>1.0.0.0</VersionPrefix>
```

Comment:
Please replace this line with `<Version>1.0.0</Version>`

Reply:
will do